packages/office-ui-fabric-react/src/components/DetailsList/DetailsColumn.base.tsx

```diff
@@ -21,13 +21,31 @@ const CLASSNAME_ADD_INTERVAL = 20; // ms
  *
  * {@docCategory DetailsList}
  */
-export class DetailsColumnBase extends BaseComponent<IDetailsColumnProps> {
+export class DetailsColumnBase extends React.Component<IDetailsColumnProps> {
+  private _async: Async;
+  private _events: EventGroup;
```

Comment:
Just a question: The reason we need this is because we're changing these components from `BaseComponent` to `React.Component` right?

Reply:
Yes. This part is from a cherry-pick of #9874,